src/linagora.esn.unifiedinbox/app/components/composer/body-editor/text/composer-body-editor-text.controller.js

```diff
@@ -1,4 +1,4 @@
-require('../../../../../../esn.inbox.libs/app/services/email-body/email-body.js');
+require('esn.inbox.libs/app/services/email-body/email-body.js');
```

Comment:
No need this to require services from external angular module `esn.inbox.libs` since you already require it in app.module.js file.

Fix it and in other files